packages/skip-button/package.json

```diff
@@ -16,6 +23,11 @@
     "dist/*",
     "scss/*"
   ],
+  "keywords": [
+    "video.js",
+    "player",
+    "videojs-plugin"
```

Comment:
Is it possible to add the component name to the keywords? if it make sens of course the cheval.

Reply:
C'est fait !